lib/backports/2.7.0/array/intersection.rb

```diff
@@ -0,0 +1,5 @@
+class Array
```

Comment:
I think we need to require `backports/1.8.7/enumerable/inject`

Reply:
Fixed. 
Two funny things:
1. I am using Ruby since, IIRC, 1.6, but I managed to forget how many things weren't here...
2. This implementation is copy-pasted from `2.6.0/array/difference.rb`, which, probably, also lacks `require`-ing of `inject` (but seems nobody complained for a year :trollface:). Should I fix it, too? What about `2.4.0/enumerable/sum.rb`?

Reply:
Good catch. I've fixed them, thanks